index.md

```diff
@@ -1,23 +1,63 @@
-_“Best practice templates and writing instructions for documenting open source software.”_
+TheGoodDocsProject* exists because every project deserves good docs, and we believe that templates make good docs. 
 
-Every project deserves a minimum level of _Good Docs_ that help users understand and solve problems with your product. But it can be really hard to know where to start or what to write. How do you structure docs? What does a docs website look like? How do I keep docs consistently formatted? The list of questions far outweigh the answers.
+At Write The Docs meetups and conferences, developers frequently ask where to get started, or what kind of documentation to produce for a new project. 
 
-That’s where _The Good Docs Project_ fits in. Our open-source Minimum Viable Docset (MVD) will help you create a baseline set of _Good Docs_ using a suite of templates. 
+We want to unblock developers (and even technical writers) by providing the necessities for documenting open source software. 
 
-Your _Good Docs_ can then be extended into _Better Docs_ with greater depth, quality and maturity.
+* Project outlines
+* Templates 
+* Best practices
 
-### Why?
+## Project outlines
```

Comment:
I think the way @cwcromwell frames the project up to this point is excellent.

After this it might be getting a bit to verbose for the scope of a README. In saying that, I definitely don't want to lose this content. 

I wonder instead of going into the verbose descriptions of outlines, templates, and best practices, we give the "twitter description" of them at this level, and then put the meat into another file?